fabfile.py

```diff
@@ -36,17 +36,17 @@ def build_static():
     local("./node_modules/.bin/gulp")
 
 
-def clear_memcached():
+def clear_redis_cache():
     try:
-        cache.init(config.MEMCACHED_SERVERS)
+        cache.init(config.REDIS_HOST, config.REDIS_PORT, config.REDIS_NAMESPACE)
```

Comment:
Good question. For now just set it up in the config file as `os.path.join(os.path.dirname(os.path.realpath(__file__)), 'cache_namespaces')`. In the near future with docker this will have to go in a mounted volume somewhere.